### ISSUE TITLE
Can't upload talk

### ISSUE THREAD

**Issue by myjeeed** [makeabilitylab/makeabilitylabwebsite#184]
Unfortunately I get this error when I want to upload my talk!

![image](https://cloud.githubusercontent.com/assets/9346800/18922728/d6b1c2b6-8577-11e6-8257-19de999f7bab.png)

Does anybody know can I fix this?

**Comment by jonfroehlich** [comment]
I just emailed UMIACS staff to see if they can help us. I can't seem to figure this one out on my own. My knowledge of unix, gunicorn, etc. is too limited. I can't get the Django log file to work on the production server (only on my local machine) and I don't know how to setup log files for gunicorn, etc. so it is extremely difficult to track down the source of this problem.

**Comment by jonfroehlich** [comment]
We have now, I think, fixed the problem. There were two issues:

1. Large files were causing a timeout on gunicorn, as we suspected. We increased the timeout from, I think, ~30s to 5mins (UMIACS warns us that this long of a timeout makes us more vulnerable to DDOS attacks but I'm really not too worried about that for our lil site).

2. We actually ran out of storage space.

I'll post here again with more details when I get a chance and then close this Issue.
